src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -700,6 +704,97 @@ private string DebuggerDisplayMethodDescription
             }
         }
 
+        #region EventSource Tracing
+
+        void EtwTaskStarted(Task previousTask, ref Guid savedActivityID)
+        {
+            TplEtwProvider etwLog = TplEtwProvider.Log;
```

Comment:
As long as we're touching all of this event-related code and moving stuff around, can we rename TplEtwProvider to instead be TplEventSource?

Reply:
Right.

Reply:
@stephentoub I am fine with rename but would prefer to do it once all code using it is in shared

Reply:
Ok